drakrun/drakrun/main.py

```diff
@@ -162,6 +162,11 @@ def generate_plugin_cmdline(self, quality, enabled_plugins):
             plugin_list = self.active_plugins[quality]
 
         plugin_list = list(set(plugin_list) & set(enabled_plugins))
+
+        if "ipt" in plugin_list and "codemon" not in plugin_list:
+            self.log.info("Using ipt plugin implies using codemon")
```

Comment:
Can we add same check on drakcore frontend to?

Reply:
@kscieslinski Do you mean displaying a warning box with the content as above?

Reply:
Yes.

Reply:
Added

![image](https://user-images.githubusercontent.com/6976513/115241075-b2757680-a120-11eb-9c61-b46af7637f49.png)